scripts/build.sh

```diff
@@ -52,44 +37,19 @@ rm -rf pkg/*
 mkdir -p bin/
 
 # Build!
-# If GOX_PARALLEL_BUILDS is set, it will be used to add a "-parallel=${GOX_PARALLEL_BUILDS}" gox parameter
 echo "==> Building..."
-gox \
-    -osarch="${XC_OSARCH}" \
+${GO_CMD} build \
     -gcflags "${GCFLAGS}" \
-    -ldflags "${LD_FLAGS}-X github.com/hashicorp/vault/sdk/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}' -X github.com/hashicorp/vault/sdk/version.BuildDate=${BUILD_DATE}" \
-    -output "pkg/{{.OS}}_{{.Arch}}/vault" \
-    ${GOX_PARALLEL_BUILDS+-parallel="${GOX_PARALLEL_BUILDS}"} \
-    -tags="${BUILD_TAGS}" \
-    -gocmd="${GO_CMD}" \
+    -ldflags "${LD_FLAGS} -X github.com/hashicorp/vault/sdk/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}' -X github.com/hashicorp/vault/sdk/version.BuildDate=${BUILD_DATE}" \
+    -o "bin/vault" \
```

Comment:
This bit will probably be missed:

```
    rm -f ${MAIN_GOPATH}/bin/vault
    cp ${F} ${MAIN_GOPATH}/bin/
```

Can you do something equivalent?  People are used to the newly built being in their path. 
 Also the rm is important on macos, something to do with their binary scanning/validation.

Reply:
`go build` does this under the hood already: https://github.com/golang/go/commit/cf7aa585ac8b3a2db8e0792d6d7c14259568823e (thanks to mitchellh for the link).

I'll re-add the bit to copy into the GOPATH.

Reply:
It does that for the build target, but if we're copying into GOPATH, *that* binary will still be vulnerable to the same problem, I think.  Happy to test it once you've made the change if you like.

Reply:
I kept the `rm` for the copy into GOPATH. It should be committed now if you would like to test.

Reply:
Yeah, I just tested with a manual `cp`, and it gets sigkilled.

Reply:
Tested your branch, works!